templates/csharp/templatedcontrol/NewTemplatedControl.axaml

```diff
@@ -1,16 +1,26 @@
-<Styles xmlns="https://github.com/avaloniaui"
+<ResourceDictionary xmlns="https://github.com/avaloniaui"
         xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
         xmlns:controls="using:AvaloniaAppTemplate.Namespace">
+
+  <!--
+    If you want support Theme variant see 
+    Docs:   https://docs.avaloniaui.net/docs/basics/user-interface/styling/control-themes
+           https://docs.avaloniaui.net/docs/guides/styles-and-resources/how-to-use-theme-variants
+    Sample: https://github.com/AvaloniaUI/Avalonia.Samples/tree/main/src/Avalonia.Samples/CustomControls/RatingControlSample#step-9-theme-variant
```

Comment:
```suggestion
    Additional resources 
    Using Control Themes:
         https://docs.avaloniaui.net/docs/basics/user-interface/styling/control-themes
    Using Theme Variants:
         https://docs.avaloniaui.net/docs/guides/styles-and-resources/how-to-use-theme-variants
```

Because sample links can be changed, but documentation shouldn't - we actively try to preserve documentation links or at least keep redirects.